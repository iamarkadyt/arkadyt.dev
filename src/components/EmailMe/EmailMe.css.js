import palette from '../../misc/colorPalette';
import { NAVBAR_HEIGHT } from '../NavBar/NavBar.css';

let styles = {
    container: {
        backgroundColor: palette.COLOR_LIGHT,
        height: window.innerHeight - NAVBAR_HEIGHT,
        display: 'flex',
        alignItems: 'center',
        justifyContent: 'center',
    },
    paragraph: {
        margin: 'auto',
        color: palette.FONT_WHITE,
        fontSize: '1.6em',
    },
    forms: {
        margin: 'auto',
    },
    fieldset: {
        border: 0,
        margin: 0,
        padding: '12px 0',
    },
    textInput: {
        height: '40px',
        width: '250px',

        borderRadius: '10px',
        border: 0,
        padding: '10px',

        fontSize: '2em',
    },
    submitButton: {
        height: '50px',
        width: '160px',
        fontSize: '1.6em',
        color: palette.FONT_WHITE,
        border: '2px solid #070',
        backgroundColor: '#0b0',
        borderRadius: '10px',
        float: 'right',
        textTransform: 'uppercase'
    },

    print() {
        console.log(this.textInput)
    }
}

styles.textInputWithMargin = {
    ...styles.textInput,
    marginLeft: '25px'
}

styles.textInputLong = {
    ...styles.textInput,
    width: '545px'
}

styles.textArea = {
    ...styles.textInput,
    width: '545px', 
    height: '200px'
}

export default styles;