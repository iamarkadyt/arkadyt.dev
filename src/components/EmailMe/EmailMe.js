import React from 'react';

class EmailMe extends React.Component {
    render() {
        return <div>
            This is emailme Component.
        </div>;
    }
}

export default EmailMe;