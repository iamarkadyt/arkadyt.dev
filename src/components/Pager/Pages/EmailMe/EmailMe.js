import React from 'react';

import classes from './EmailMe.module.css';

import SMLinks from './SMLinks/SMLinks';

const emailMe = () => {
    return (
        <div className={classes.container}>
            <p>Aliquip est sit elit eiusmod quis consequat. Elit Lorem minim exercitation nulla ut reprehenderit ea. Ea ut aute Lorem sint dolor et nostrud qui non irure ullamco aliquip esse eiusmod. Laborum minim pariatur irure laborum ex minim consequat laboris esse proident est laboris. Laborum in minim ipsum consectetur cupidatat proident eu commodo voluptate enim. Qui aliquip ipsum excepteur sunt consectetur aliqua voluptate aliqua do officia duis anim.</p>
            <form>
                <fieldset>
                    <input type='text' placeholder='Your name' />
                </fieldset>
                <fieldset>
                    <input type='email' placeholder='Email' />
                </fieldset>
                <fieldset>
                    <textarea placeholder='Message' />
                </fieldset>
                <fieldset>
                    <input type='submit' value='Submit' />
                </fieldset>
            </form>
            <div style={{ gridColumn: '1/3' }}>
                <SMLinks />
            </div>
        </div>
    );
};

export default emailMe;