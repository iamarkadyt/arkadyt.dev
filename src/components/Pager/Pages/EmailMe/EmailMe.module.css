.container {
    width: 100%;
    height: 100%;

    display: grid;
    grid-template-columns: 1fr 1fr;
    justify-items: center;
    align-items: center;

    grid-auto-rows: auto 20%;
}

p {
    color: antiquewhite;
    padding: 0 2vw;
    font-family: Arial, Helvetica, sans-serif;
    font-size: 1.5em;
}

form {
    text-align: right;
    padding: 0 2vw;
}

fieldset {
    border: 0;
    margin: 1vh;
    padding: 0;
}

input[type=text], input[type=email] {
    width: 35vw;
    
    border: 0;
    border-radius: 5px;

    font-size: 30px;
    padding: 10px;
}

textarea {
    height: 150px;
    width: 35vw;
    
    border: 0;
    border-radius: 5px;

    font-size: 30px;
    padding: 10px;

    font-family: Arial, Helvetica, sans-serif;
}

input[type=submit] {
    border: 0;
    border-radius: 5px;

    font-size: 30px;
    padding: 10px 70px;

    background-color: greenyellow;
}