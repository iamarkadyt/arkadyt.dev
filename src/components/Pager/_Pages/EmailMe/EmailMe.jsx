import React from 'react';
import Card from '../../Card/Card';

import classes from './EmailMe.module.css';

const emailMe = () => {
    const text = `
        Sunt incididunt duis cillum aliquip deserunt amet adipisicing officia dolor sint laboris. Tempor culpa incididunt sint cillum laboris exercitation ullamco aliqua. Aliquip ipsum voluptate adipisicing dolore ex elit sint veniam adipisicing.
    `;
    return <Card header='Hi,' body={text}>
        <div className={classes.content}>
            <form>
                <fieldset>
                    <input type='text' placeholder='Your name' />
                </fieldset>
                <fieldset>
                    <input type='email' placeholder='Email' />
                </fieldset>
                <fieldset>
                    <textarea placeholder='Message' />
                </fieldset>
                <fieldset>
                    <input type='submit' value='Submit' />
                </fieldset>
            </form>
        </div>
    </Card>;
};

export default emailMe;