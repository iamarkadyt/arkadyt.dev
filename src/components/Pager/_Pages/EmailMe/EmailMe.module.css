.content {
    display: flex;
    justify-content: center;
    align-items: center;
}

.content form {
    text-align: left;
    width: 100%;
    margin: 0 1vw;
}

.content fieldset {
    border: 0;
    padding: 0;
}

.content input[type=text],
input[type=email] {
    width: 100%;
    padding: 5px;
    margin-bottom: 8px;
    border: 0;
    border-radius: 5px;
    font-size: 20px;
}

.content textarea {
    width: 100%;
    padding: 5px;
    margin-bottom: 4px;
    border: 0;
    border-radius: 5px;
    font-size: 20px;
    font-family: Arial, Helvetica, sans-serif;
    height: 150px;
}

.content input[type=submit] {
    background-color: greenyellow;
    padding: 5px 35px;
    border: 0;
    border-radius: 5px;
    font-size: 20px;
}