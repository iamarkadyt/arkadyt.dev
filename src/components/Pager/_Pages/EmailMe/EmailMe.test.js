import React from 'react';
import Enzyme, { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import EmailMe from './EmailMe';

configure({ adapter: new Adapter() });

it('matches snapshot.', () => {
    const tree = renderer.create(<EmailMe />).toJSON();
    expect(tree).toMatchSnapshot();
});