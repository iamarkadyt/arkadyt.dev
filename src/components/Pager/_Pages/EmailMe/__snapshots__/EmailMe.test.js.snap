// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot. 1`] = `
<div
  className="container"
>
  <div
    className="content"
  >
    <p>
      Aliquip est sit elit eiusmod quis consequat. Elit Lorem minim exercitation nulla ut reprehenderit ea. Ea ut aute Lorem sint dolor et nostrud qui non irure ullamco aliquip esse eiusmod. Laborum minim pariatur irure laborum ex minim consequat laboris esse proident est laboris. Laborum in minim ipsum consectetur cupidatat proident eu commodo voluptate enim. Qui aliquip ipsum excepteur sunt consectetur aliqua voluptate aliqua do officia duis anim.
    </p>
    <form>
      <fieldset>
        <input
          placeholder="Your name"
          type="text"
        />
      </fieldset>
      <fieldset>
        <input
          placeholder="Email"
          type="email"
        />
      </fieldset>
      <fieldset>
        <textarea
          placeholder="Message"
        />
      </fieldset>
      <fieldset>
        <input
          type="submit"
          value="Submit"
        />
      </fieldset>
    </form>
  </div>
</div>
`;
